tests/unit/test_main.py

```diff
@@ -1,31 +1,35 @@
 from fastapi.testclient import TestClient
 import json
 import base64
-
+import sys
+try:
+    sys.path.insert(1, 'src/')
+except:
+    pass
```

Comment:
What happens when `sys.path.insert(1, 'src/')` fails? Do you think we should write a exception better than `pass`.
```suggestion
    print('Python path can't be set')
```

Reply:
It will fail when we run pytest inside docker because in the docker src folder is not there. In local environment, it only works in the root directory.